tools/topology/sof-cml-demux-rt5682-max98357a.m4

```diff
@@ -53,7 +53,7 @@ PCM_PLAYBACK_ADD(Speakers, 6, PIPELINE_PCM_8)
 #SSP 1 (ID: 6)
 DAI_CONFIG(SSP, 1, 6, SSP1-Codec,
 	SSP_CONFIG(I2S, SSP_CLOCK(mclk, 24000000, codec_mclk_in),
-		SSP_CLOCK(bclk, 1500000, codec_slave),
-		SSP_CLOCK(fsync, 46875, codec_slave),
-		SSP_TDM(2, 16, 3, 3),
-		SSP_CONFIG_DATA(SSP, 1, 16)))
+		SSP_CLOCK(bclk, 2304000, codec_slave),
```

Comment:
Earlier topology was only used for enabling purpose. Since the m/n divider is part of the new drop now, i pushed this change. Can we merge this so that we update the new topology ?

Is there a way we can protect this change under ABI dependancy - say if ABI > x then choose this config, else move to old config ?